go/go.mod

```diff
@@ -2,27 +2,27 @@ module github.com/camunda/camunda-platform-get-started/go
 
 go 1.17
 
-require github.com/camunda/zeebe/clients/go/v8 v8.2.5
+require github.com/camunda/zeebe/clients/go/v8 v8.2.6-0.20230607111951-e3fe32126489
```

Comment:
Seems a bit strange, I figured it would just stay on 8.2.6 :thinking: 

That reminds me, we bumped the Zeebe client to 1.19, so we should probably bump it here too.